app.py

```diff
@@ -66,44 +66,88 @@ def adduser():
     return jsonify({'id' : new_user.user_id}), 201
 
 
-@app.route('/homecookedfood/addtiffin', methods=['POST'])
+@app.route('/hcf/addmeal', methods=['POST'])
 @auth.login_required
-def addtiffin():
+def addmeal():
     content = request.json
     user_id = g.user.user_id
 
-    if (not content or not 'tiffin_details' in content):
+    if (not content or not 'meal_details' in content or not price_per_meal in content):
         abort(400)
```

Comment:
nope, user can retry as many number of times as possible. Its just that this particular request cannot proceed ahead.